Oliver-ke/Teamwork
# #28595636 Implement Admin create-user feature

#### What does this PR do?
- allows admin create and employee user

#### Description of Task to be completed?
- install related dependencies
- create model for users
- add automated test for feature
- add routes for creating users

#### How should this be manually tested?
- clone this repository
- run `npm instal` to install all dependences
- ensure you have postgres and your env file setup properly
- run `npm run start:dev` to startup the development server
- open postman send POST request to `localhost:5000/api/v1/auth/create-user`
- you should see a response

#### Any background context you want to provide?
- ensure you login with admin account to get valid auth token

#### What are the relevant project board stories?
[#28595636](https://github.com/Oliver-ke/Teamwork/projects/1#card-28595636)

#### Screenshots (if appropriate)
none

#### Questions:
none
